controller/logs.go

```diff
@@ -0,0 +1,60 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+)
+
+func (c *Controller) GetActiveDeploymentLogs(ctx context.Context, numLines int32) error {
+	projectID, err := c.cfg.GetProject()
+	if err != nil {
+		return err
+	}
+	environmentID, err := c.cfg.GetEnvironment()
+	if err != nil {
+		return err
+	}
+	deployments, err := c.gtwy.GetDeploymentsForEnvironment(ctx, projectID, environmentID)
+	if err != nil {
+		return err
+	}
+
+	latestDeploy := deployments[0]
+	// Streaming
+	prevIdx := 0
+	for {
+		err := func() error {
```

Comment:
Curious what the purpose of the wrapping function here is?

Reply:
Originally, so that I can use return to trigger the deferred sleep, but I pulled it so will remove